app/views/users/edit.html.erb

```diff
@@ -0,0 +1,94 @@
+<body class="edit-view">
```

Comment:
Interesting, what motivated the switch from .haml to .erb?

Reply:
There's a lot more online support when it comes to integrating bootstrap and flexbox with .erb files!